package.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "devmountian_capstone",
+	"version": "1.0.0",
+	"description": "",
+	"main": "calculator.js",
+	"scripts": {
+		"start": "http-server ./"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC",
+	"devDependencies": {
+		"http-server": "^14.1.1"
+	}
+}
```

Comment:
- Added a package.json file by doing `npm init -y` to generate a default file.
- Installed the [http-server](https://www.npmjs.com/package/http-server) npm module to reference from our start script.
- Added a `"start"` script to the package.json file that will invoke `http-server` on the current directory, hosting the files at `http://localhost:8080`.